README.md

```diff
@@ -64,7 +64,13 @@ English resources about 'war' and/or 'peace':
 
 ### Sorting
 
-All search queries support `sort`ing on `title` or `date`. To set a non-default direction use `sort_direction=(asc|desc)`. To sort by relevance, omit the `sort` param.
+All search queries support `sort`ing on:
+
+ - `title`: Case insensitive sort on title. Default ascending.
+ - `date`: Sort on dateStartYear. Default descending.
+ - `creator`: Case insensitive sort on first creator. (Note the "first" creator may not be the best creator.) Default ascending.
```

Comment:
For the record, I'm not sure Creator is useful to sort by - particularly for queries with lots of results - but it was a request from a long time ago, so the index field exists for it. It's avail to use or not use.